ansible_collections/arista/avd/roles/eos_designs/doc/fabric-topology.md

```diff
@@ -333,7 +333,18 @@ default_interfaces:
 < node_type_key >:
 
   defaults:
-    # node BGP AS | Required.
+    # Node BGP AS | Required.
+    # For eBGP topologies, bgp_as can be defined as a range of ASNs under defaults.  This range will be expanded and ASNs allocated
+    # according to the following rules:
+    #  - Standalone nodes and Non-MLAG (i.e. EVPN A/A multi-homing) node groups will be allocated unique ASNs from the range based on
+    #    the node's ID.
+    #  - Switches in MLAG pairs will be allocated a single ASN, based on the node ID of the first node in the node group.
+    # Examples:
+    # bgp_as: 65101-65110                 Contiguous range of ASNs.
+    # bgp_as: 65001,65010,65020,65030     Non-contiguous range of ASNs.
+    # bgp_as: 65000.101-125               Contiguous range of 4-byte ASNs, increment in the last 2 bytes.
+    # bgp_as: 64512                       Single AS, no range.
+    # Commas and hyphens can be combined in more-or-less any combination.
```

Comment:
not a native English speaker here so just a question regarding hyphenated `more-or-less` - is it supposed to be spelled like this?

Reply:
So I just looked this up and I think this is just a habitual thing of mine.  Will remove.